src/scss/pages/carousel.scss

```diff
@@ -234,16 +241,33 @@
   }
 }
 @media (max-width: 768px) {
+  .carousel-container {
+    height: 300px;
+    .section__title{
+    font-size: 3.0em;
+    padding-bottom: 5%;
+  }
+}
+
   .alice-carousel__stage-item img {
    position: relative;
    right: -63px;
+  //  height: 97px;
```

Comment:
look, if you comment on some code and you need to leave it for some reason, you should also leave a comment describing this reason. Otherwise, remove any comments